sapi/fpm/fpm/fpm_sockets.c

```diff
@@ -39,6 +39,16 @@ static struct fpm_array_s sockets_list;
 
 enum { FPM_GET_USE_SOCKET = 1, FPM_STORE_SOCKET = 2, FPM_STORE_USE_SOCKET = 3 };
 
+static inline void fpm_sockets_env_set(char *envname, unsigned idx) /* {{{ */
```

Comment:
```suggestion
static inline void fpm_sockets_get_env_name(char *envname, unsigned idx) /* {{{ */
```
Maybe? I found the name of the function a bit confusing, as it does not actually set the environment variable.

Reply:
yeah that makes more sense - renamed.